tests/CSSList/DocumentTest.php

```diff
@@ -85,4 +85,66 @@ public function setContentsReplacesContentsSetInPreviousCall()
 
         self::assertSame($contents2, $this->subject->getContents());
     }
+
+    /**
+     * @return \Generator<array<array<DeclarationBlock>, DeclarationBlock, DeclarationBlock, array<DeclarationBlock>>>
```

Comment:
I think my previous comment about the array notation was misleading and incorrect. Please have a look at the PHPStan documentation on this:

https://phpstan.org/writing-php-code/phpdoc-types#general-arrays
https://phpstan.org/writing-php-code/phpdoc-types#array-shapes

Reply:
I hope I got it right this time.

Reply:
Also, I switched it back to a simple array as I do not see the additional benefit of using a Generator in this case.